datastream-server/src/main/java/com/linkedin/datastream/server/assignment/StickyMulticastStrategy.java

```diff
@@ -59,13 +60,17 @@
  * </ul>
  */
 public class StickyMulticastStrategy implements AssignmentStrategy {
+  public static final String CFG_MIN_TASKS = "minTasks";
```

Comment:
As discussed offline, I've created a separate method to just calculate the number of tasks, and removed all the elastic task stuff from the `StickyMulticastStrategy`. This way, it can only be enabled for `StickyPartitionAssignmentStrategy` users.